docs/src/formulation.md

```diff
@@ -50,19 +50,32 @@ In addition, the following flow sets represent methods for incorporating additio
 | $\mathcal{K}$     | Representative periods (rp)                         | $k \in \mathcal{K}$       | $\mathcal{K} \subset \mathbb{N}$ | $\mathcal{K}$ does not have to be a subset of $\mathcal{P}$                |
 | $\mathcal{B}_{k}$ | Timesteps blocks within a representative period $k$ | $b_{k} \in \mathcal{B}_k$ |                                  | $\mathcal{B}_k$ is a partition of timesteps in a representative period $k$ |
 
+### Sets for Groups
+
+| Name                     | Description             | Elements                       | Superset | Notes |
+| ------------------------ | ----------------------- | ------------------------------ | -------- | ----- |
+| $\mathcal{G}^{\text{a}}$ | Groups of energy assets | $g \in \mathcal{G}^{\text{a}}$ |          |       |
+
+In addition, the following subsets represent methods for incorporating additional constraints in the model.
+
+| Name                      | Description                                         | Elements | Superset                                                   | Notes                                                                                                                                                                          |
+| ------------------------- | --------------------------------------------------- | -------- | ---------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| $\mathcal{G}^{\text{ai}}$ | Group of assets that share min/max investment limit |          | $\mathcal{G}^{\text{ai}} \subseteq \mathcal{G}^{\text{a}}$ | This set contains a group of investment assets that use investment method. Please visit the [how-to section](@ref investment-group-setup) to learn how to set up this feature. |
```

Comment:
"that use SOMETHING investment method"

An investment method? The same investment method?

Reply:
Ah now I think it should be "This set contains assets that have a group investment limit." Or something like that.

Reply:
Done! Thanks